src/Modulo2.jl

```diff
@@ -663,6 +679,16 @@ function copyto!(a::ZZ2Array{N}, bc::Broadcasted{ZZ2ArrayStyle{N}, <:Any, typeof
     a
 end
 
+#
+# Utility functions
+#
+
+count_ones(a::ZZ2) = Int(isodd(a))
+count_zeros(a::ZZ2) = Int(iseven(a))
+# TODO: use views to not reduce over the padding to multiples of 256?
```

Comment:
The whole point of the padding to make things faster. I think there is no benefit in using views.

Reply:
To be clear, I meant to retain the padding to 64bit chunks, and only some over `@view a.data[1:n]` for `n = div(a.i1,64)+1`. Perhaps the hassle of checking the right length outweighs the benefits. There might not be any benefit if `count_ones` etc SIMDs over the 4 chunks anyway, but I did not check that.